sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/StringUtils.scala

```diff
@@ -81,7 +81,7 @@ object StringUtils extends Logging {
   /**
    * This utility can be used for filtering pattern in the "Like" of "Show Tables / Functions" DDL
    * @param names the names list to be filtered
-   * @param pattern the filter pattern, only '*' and '|' are allowed as wildcards, others will
+   * @param pattern the filter pattern, only '%', '_' and '|' are allowed as wildcards, others will
```

Comment:
Do we need to add a test for the new wildcard characters supported?

Reply:
> Do we need to add a test for the new wildcard characters supported?

emmm, I am waiting current but test result and fix current failed unit test and add test for new wildcard.